website/docs/r/cloned_vapp.html.markdown

```diff
@@ -0,0 +1,80 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_cloned_vapp"
+sidebar_current: "docs-vcd-resource-cloned-vapp"
+description: |-
+  Provides a VMware Cloud Director Cloned vApp resource. This can be used to create vApps from either a vApp template or another vApp.
+---
+
+# vcd\_cloned\_vapp
+
+Provides a VMware Cloud Director Cloned vApp resource. This can be used to create vApps from either a vApp template or another vApp.
+This resource should be used only on creation, although deletion also works. The result of using this resource is a
```

Comment:
Just a thought, maybe add a sentence on the main targeted use case, as it may not be instantly clear in what circumstances one would use it?

Reply:
Added

Reply:
Thanks! BTW, is there a scenario where user would want to import the cloned vApp with all its VMs to Terraform management? It feels like this could be a separate section in this doc page.

Reply:
I am sure users would love it, but cross-resource operations are not a good fit for Terraform.
The point is that this resource like `vcd_subscribed_catalog` only handles the main resource, and the byproducts can only be handled by separate use of data sources and imported simpler resources.
We may find some partial solution to this problem in a later release, where we will apply the new import infrastructure.